api/src/main/java/org/openmrs/module/fhir/api/util/StrategyUtil.java

```diff
@@ -0,0 +1,17 @@
+package org.openmrs.module.fhir.api.util;
+
+import org.hl7.fhir.dstu3.model.DomainResource;
+import org.hl7.fhir.dstu3.model.IdType;
+
+import java.util.List;
+
+public class StrategyUtil {
```

Comment:
This class can be final and should have private constructor.

Reply:
Ok